Formula/rust.rb

```diff
@@ -3,8 +3,8 @@ class Rust < Formula
   homepage "https://www.rust-lang.org/"
 
   stable do
-    url "https://static.rust-lang.org/dist/rustc-1.43.1-src.tar.gz"
-    sha256 "cde177b4a8c687da96f20de27630a1eb55c9d146a15e4c900d5c31cd3c3ac41d"
+    url "https://static.rust-lang.org/dist/rustc-1.44.0-src.tar.gz"
+    sha256 "bf2df62317e533e84167c5bc7d4351a99fdab1f9cd6e6ba09f51996ad8561100"
 
     resource "cargo" do
       url "https://github.com/rust-lang/cargo.git",
```

Comment:
This resource should also be bumped to 0.45.0.

Reply:
Augh, the Cargo 0.45.0 release didn't exist yet when I created the PR, unfortunately. I'll have to remember to wait a bit in the future. Thanks for the catch.

I'm going to kill this CI run for now and I'll restart it once the `oxipng` issue is resolved.